src/main/java/com/iota/iri/conf/TipSelConfig.java

```diff
@@ -8,17 +8,23 @@
 public interface TipSelConfig extends Config {
 
     /**
-     * @return Descriptions#MAX_DEPTH
+     * Default Value: {@value BaseIotaConfig.Defaults#MAX_DEPTH}
+     *
+     * @return {@value TipSelConfig.Descriptions#MAX_DEPTH}
      */
     int getMaxDepth();
 
-     /**
-     * @return Descriptions#ALPHA
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#ALPHA}
+     *
+     * @return {@value TipSelConfig.Descriptions#ALPHA}
      */
     double getAlpha();
 
     /**
-     * @return Descriptions#BELOW_MAX_DEPTH_TRANSACTION_LIMIT
+     * Default Value: {@value BaseIotaConfig.Defaults#MAX_ANALYZED_TXS}
```

Comment:
the default value name does not match the description name.
`MAX_ANALYZED_TXS` vs `BELOW_MAX_DEPTH_TRANSACTION_LIMIT`